tests/twig-check/composer.json

```diff
@@ -0,0 +1,14 @@
+{
+  "require-dev": {
+    "vincentlanglet/twig-cs-fixer": "^3.1"
+  },
+  "config": {
+    "platform": {
+      "php": "8.2"
+    }
+  },
+  "scripts": {
+    "twig": "vendor/bin/twig-cs-fixer lint \"../../views/templates\"",
+    "twig:fix": "vendor/bin/twig-cs-fixer fix \"../../views/templates\""
+  }
+}
```

Comment:
It's starting to have a lot of composer.json... We have 4 including the storybook one. We don't have a way to use the main composer, with the dependency in requires-dev?

Reply:
I tried using the main composer.json with the dependencies in require-dev, but I couldn’t get it to work because it requires a more recent PHP version. Do we have any other options?

Reply:
I think we could use docker. You must create an image, use the PHAR given by the project

Ex: 

```
FROM php:8.2-cli

RUN wget -c https://github.com/VincentLanglet/Twig-CS-Fixer/releases/download/3.1.0/twig-cs-fixer.phar \
    -O /usr/local/bin/twig-cs-fixer

RUN chmod +x /usr/local/bin/twig-cs-fixer

ENTRYPOINT ["twig-cs-fixer"]
CMD ["twig"]

```
`docker build -t twig-cs-fixer-image .`
`docker run --rm twig-cs-fixer-image`

Reply:
If there is no way to have an additional tool in our existing dependency list, I would rather not try to force its inclusion in the project because it would create one more way to run a specific binary. :/

Reply:
Ok, I close this PR !